projects/helper/coreAssets.json

```diff
@@ -1306,7 +1306,7 @@
     "USDT": "0x068f5c6a61780768455de69077e07e89787839bf8166decfbf92b645209c0fb8",
     "WBTC": "0x03fe2b97c1fd336e750087d68b9b867997fd64a2661ff3ca5a7c771641e8e7ac",
     "ETH": "0x049d36570d4e46f48e99674bd3fcc84644ddd6b96f7c741b1562b82f9e004dc7",
-    "WSTETH": "0x042b8f0484674ca266ac5d08e4ac6a3fe65bd3129795def2dca5c34ecc5f96d2",
+    "WSTETH_Legacy": "0x042b8f0484674ca266ac5d08e4ac6a3fe65bd3129795def2dca5c34ecc5f96d2",
```

Comment:
please revert this, changing keys will have side effects on other projects using the same key

Reply:
ok, so there now exist two wstETH tokens on starknet bc StarkGate, the canonical bridge, migrated the legacy one to a new implementation https://github.com/DefiLlama/DefiLlama-Adapters/pull/13558/files#diff-d2d52b699b64d1224d64e4f8017ab0bc41869eafd84c153530d5b0fdeb08f34cR1319

should I rename the new implementation something like "WSTETH_1"? if I do this is there a way to label the old one with something like "wstETH (legacy)" (which is not the official token symbol) and have the new implementation use the "wstETH" symbol? would be confusing otherwise.

Reply:
nvm I fixed the keys